scripts/ci/travis_build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+if [ -n "${TRAVIS_TAG}" ]; then
+  APP_NAME="QField"
+  PKG_NAME="qfield"
+  APP_ICON="qfield-logo.svg"
+  APP_VERSION="${TRAVIS_TAG}"
+elif [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then
+  APP_NAME="QField Nightly"
+  PKG_NAME="qfield_nightly"
+  APP_ICON="qfield-testlogo.svg"
+  APP_VERSION="Nightly ${TRAVIS_COMMIT}"
+else
+  APP_NAME="QField Beta ${TRAVIS_PULL_REQUEST}"
+  PKG_NAME="qfield_beta"
+  APP_ICON="qfield-testlogo.svg"
+  APP_VERSION="Pull Request ${TRAVIS_PULL_REQUEST}"
+fi
+docker run -v $(pwd):/usr/src/qfield -e "BUILD_FOLDER=build-${ARCH}" -e "ARCH=${ARCH}" -e "STOREPASS=${STOREPASS}" -e "KEYNAME=${KEYNAME}" -e "KEYPASS=${KEYPASS}" -e "VERSION=${APP_VERSION}" -e "PKG_NAME=${PKG_NAME}" -e "APP_NAME=${APP_NAME}" -e "APP_ICON=${APP_ICON}" opengisch/qfield-sdk:${QFIELD_SDK_VERSION} /usr/src/qfield/scripts/docker-build.sh
```

Comment:
use an env file?

or at least do not repeat variables if they are taken as such:
```suggestion
docker run -v $(pwd):/usr/src/qfield -e "BUILD_FOLDER=build-${ARCH}" -e ARCH -e STOREPASS -e KEYNAME -e KEYPASS -e "VERSION=${APP_VERSION}" -e PKG_NAME -e APP_NAME -e APP_ICON opengisch/qfield-sdk:${QFIELD_SDK_VERSION} /usr/src/qfield/scripts/docker-build.sh
```

Reply:
What would be the advantage of an env file? I only see that it would be harder to pass in dynamic content (or not?)

I'll accept this proposal meanwhile ;)

Reply:
just makes it easier to read/modify than these oneliners